validator_derive/tests/range.rs

```diff
@@ -1,5 +1,12 @@
 use validator::{Validate, ValidationErrors};
 
+// Loose floating point comparison using EPSILON error bound
+macro_rules! assert_float {
```

Comment:
This fixes an unrelated-to-this-PR clippy lint about literal float comparisons. It's encapsulated in the second commit on this PR. I'm happy to rip that commit out if we don't want to include that in this.

Reply:
no that's good